### PR TITLE
chore: update names in data schema

### PR DESCRIPTION
**Pull request by kjappelbaum**
Similar to what we discussed. Added also gerund ("ing form"). 
Starting adjusting some examples, will continue and adjust contribution guide upon first feedback.

### CODE REVIEW

File: data/caco2_wang/transform.py [hunk 1]
```diff
@@ -59,9 +59,9 @@ def get_and_transform_data():
                 "units": "cm/s",
                 "type": "continuous",  # can be "categorical", "ordinal", "continuous"
                 "names": [  # names for the property (to sample from for building the prompts)
-                    "Caco-2 cell effective permeability",
-                    "Caco-2 cell permeability",
-                    "Caco-2 permeability",
+                    {"noun": "Caco-2 cell effective permeability"},
```
Comment (MicPie) [inline data/caco2_wang/transform.py:62]: I was thinking about having a list under the `noun` key, but both would work

File: data/cav3_t-type_calcium_channels_butkiewicz/transform.py [hunk 2]
```diff
@@ -51,9 +51,9 @@ def get_and_transform_data():
                 "units": None,
                 "type": "boolean",
                 "names": [
-                    "a inhibitor of cav3 t-type calcium channels activity",
-                    "inhibiting cav3 t-type calcium channels activity",
-                    "a t-type calcium channel blocker",
+                    {"noun": "an inhibitor of cav3 t-type calcium channels activity"},
+                    {"gerund": "inhibiting cav3 t-type calcium channels activity"},
```
Comment (MicPie) [inline data/cav3_t-type_calcium_channels_butkiewicz/transform.py:55]: the addition of `gerund` is a great idea!

**Review by MicPie — APPROVED** [review]

File: data/lipophilicity/transform.py [hunk 3]
```diff
@@ -45,8 +45,9 @@ def get_and_transform_data():
                 "units": None,
                 "type": "continuous",  # can be "categorical", "ordinal", "continuous"
                 "names": [  # names for the property (to sample from for building the prompts)
-                    "octanol/water distribution coefficient (logD at pH 7.4)",
-                    "octanol/water distribution coefficient",
+                    {"name": "octanol/water distribution coefficient (logD at pH 7.4)"},
```
Comment (MicPie) [inline data/lipophilicity/transform.py:48]: name --> noun?

File: data/m1_muscarinic_receptor_agonists_butkiewicz/transform.py [hunk 4]
```diff
@@ -56,8 +56,12 @@ def get_and_transform_data():
                 "units": None,
                 "type": "boolean",
                 "names": [
-                    "a positive allosteric modulator of m1 muscarinic activity",
-                    "displaying positive allosteric modulation of the m1 muscarinic receptor",
+                    {
+                        "name": "a positive allosteric modulator of m1 muscarinic activity"
```
Comment (MicPie) [inline data/m1_muscarinic_receptor_agonists_butkiewicz/transform.py:60]: name --> noun?

File: data/lipophilicity/transform.py [hunk 5]
```diff
@@ -45,8 +45,9 @@ def get_and_transform_data():
...
+                    {"name": "logD at pH 7.4"},
+                    {"name": "octanol/water distribution coefficient"},
                 ],
                 "uris": [
                     "http://www.bioassayontology.org/bao#BAO_0002129",
```
Comment (MicPie) [inline data/lipophilicity/transform.py:53]: Yaml needs to be recreated?

File: data/m1_muscarinic_receptor_agonists_butkiewicz/transform.py [hunk 6]
```diff
@@ -56,8 +56,12 @@ def get_and_transform_data():
...
+                    },
+                    {
+                        "gerund": "displaying positive allosteric modulation of the m1 muscarinic receptor"
+                    },
                 ],
                 "pubchem_aids": [626, 1488, 1741],
                 "uris": [],
```
Comment (MicPie) [inline data/m1_muscarinic_receptor_agonists_butkiewicz/transform.py:67]: Yaml needs to be recreated?

File: data/m1_muscarinic_receptor_antagonists_butkiewicz/transform.py [hunk 7]
```diff
@@ -46,8 +46,8 @@ def get_and_transform_data():
                 "units": None,
                 "type": "boolean",
                 "names": [
-                    "a negative modulator of M1 muscarinic receptors",
-                    "negatively modulating M1 muscarinic receptors",
+                    {"noun": "a negative modulator of M1 muscarinic receptors"},
+                    {"gerund": "negatively modulating M1 muscarinic receptors"},
                 ],
                 "pubchem_aids": [628, 677, 860],
                 "uris": [],
```
Comment (MicPie) [inline data/m1_muscarinic_receptor_antagonists_butkiewicz/transform.py:53]: Yaml needs to be recreated?

File: data/orexin1_receptor_butkiewicz/transform.py [hunk 8]
```diff
@@ -46,9 +46,9 @@ def get_and_transform_data():
...
                 "names": [
-                    "is a orexin 1 inhibitor",
-                    "is a orexin 1 receptor antagonist",
-                    "inhibits orexin 1 receptor",
+                    {"noun": "orexin 1 inhibitor"},
+                    {"noun": "a orexin 1 receptor antagonist"},
+                    {"gerund": "inhibiting orexin 1 receptor"},
                 ],
                 "pubchem_aids": [485270, 463079, 434989, 504701, 493232, 504699],
                 "uris": ["http://purl.bioontology.org/ontology/SNOMEDCT/838464006"],
```
Comment (MicPie) [inline data/orexin1_receptor_butkiewicz/transform.py:54]: Yaml needs to be recreated?

File: data/pampa_ncats/transform.py [hunk 9]
```diff
@@ -47,8 +47,11 @@ def get_and_transform_data():
...
-                    "is permeable in the PAMPA assay",
-                    "shows permeability in parallel artificial membrane permeability assay (PAMPA) assay",
+                    {"verb": "is permeable in the PAMPA assay"},
+                    {
+                        "verb": "shows permeability in parallel artificial membrane permeability assay (PAMPA) assay"
+                    },
+                    {"gerund": "permeating in the PAMPA assay"},
                 ],
                 "pubchem_aids": [1508612],
                 "uris": ["http://purl.bioontology.org/ontology/MESH/D002463"],
```
Comment (MicPie) [inline data/pampa_ncats/transform.py:57]: Yaml needs to be recreated?

File: data/potassium_ion_channel_kir2_1_butkiewicz/transform.py [hunk 10]
```diff
@@ -57,9 +57,12 @@ def get_and_transform_data():
...
-                    "inhibts the I  nward-Rectifying Potassium Ion Channel Kir2.1",
+                    {"verb": "is blocking potassium ion channel activity"},
+                    {"verb": "blocks potassium ion channel activity"},
+                    {
+                        "verb": "inhibts the Inward-Rectifying Potassium Ion Channel Kir2.1"
+                    },
+                    {"noun": "blocker of potassium ion channel activity"},
                 ],
                 "pubchem_aids": [1672, 2032, 463252, 2105, 2345, 2236, 2329],
                 "uris": ["http://purl.obolibrary.org/obo/XCO_0000225"],
```
Comment (MicPie) [inline data/potassium_ion_channel_kir2_1_butkiewicz/transform.py:68]: Yaml needs to be recreated?

**Review by MicPie — CHANGES_REQUESTED** [review]

File: data/flashpoint/meta.yaml [hunk 11]
```diff
@@ -1,37 +1,37 @@
 ---
```
[the remark this answers is not in the document]
Comment (kjappelbaum) [reply data/flashpoint/meta.yaml:1]: not so sure. This here follows the originally anticipated pattern that the meta.yaml is read in Python

File: data/cav3_t-type_calcium_channels_butkiewicz/transform.py [hunk 12]
```diff
@@ -54,6 +54,8 @@ def get_and_transform_data():
                     {"noun": "an inhibitor of cav3 t-type calcium channels activity"},
                     {"gerund": "inhibiting cav3 t-type calcium channels activity"},
                     {"noun": "a t-type calcium channel blocker"},
+                    {"verb": "blocks t-type calcium channels"},
+                    {"verb": "inhibts cav3 t-type calcium channels"},
```
[the remark this answers is not in the document]
Comment (kjappelbaum) [reply data/cav3_t-type_calcium_channels_butkiewicz/transform.py:58]: Done, thanks

File: data/cav3_t-type_calcium_channels_butkiewicz/transform.py [hunk 13]
```diff
@@ -54,6 +54,8 @@ def get_and_transform_data():
...
                 ],
                 "pubchem_aids": [1053190, 489005, 493021, 493022, 493023, 493041],
                 "uris": ["http://purl.obolibrary.org/obo/CHEBI_194338"],
```
[the remark this answers is not in the document]
Comment (kjappelbaum) [reply data/cav3_t-type_calcium_channels_butkiewicz/transform.py:61]: Done

File: data/choline_transporter_butkiewicz/transform.py [hunk 14]
```diff
@@ -63,6 +63,7 @@ def get_and_transform_data():
                 "names": [  # names for the property (to sample from for building the prompts)
                     {"noun": "a inhibitor of choline transporter activity"},
                     {"noun": "inhibitor of choline transporter activity"},
+                    {"verb": "inhibits choline transporter activity"},
                 ],
                 "pubchem_aids": [488975, 493221, 504840, 588401, 493222, 602208],
             },
```
[the remark this answers is not in the document]
Comment (kjappelbaum) [reply data/choline_transporter_butkiewicz/transform.py:69]: Done

**Review by kjappelbaum — COMMENTED** [review]
@MicPie i think it should be ok now

File: data/m1_muscarinic_receptor_agonists_butkiewicz/transform.py (hunk 4 above)
Comment (MicPie) [reply data/m1_muscarinic_receptor_agonists_butkiewicz/transform.py:60]: @kjappelbaum maybe uncommited/unpushed change?

**Review by MicPie — CHANGES_REQUESTED** [review]

**Pull request merged**